### PR TITLE
DEVPROD-3776 Add fields to Task in evergreen.py

### PR DESCRIPTION
**Pull request by namazgurbanovmdb**

### CODE REVIEW

**Review by rongodb — CHANGES_REQUESTED** [review]
need to update the readme and the project.toml as well.

**Review by gmishkin — COMMENTED** [review]
I don't see these fields on https://docs.devprod.prod.corp.mongodb.com/evergreen/API/REST-V2-Usage#tag/tasks/paths/~1tasks~1%7Btask_id%7D/get

Aren't these fields only added by our flaky detection based on the Trino data? They are not part of the Evergreen REST API task object.

If we want a single class to store this data we need to inherit the evergreen task object and augment it, or make a wrapper class that adds the Trino fields and overrides `__call__` or something to get other fields from the evergreen.py task object.

File: src/evergreen/task.py [hunk 1]
```diff
@@ -148,6 +148,7 @@ class Task(_BaseEvergreenObject):
     display_only = evg_attrib("display_only")
     display_status = evg_attrib("display_status")
     distro_id = evg_attrib("distro_id")
+    end_time = evg_datetime_attrib("end_time")
```
[the remark this answers is not in the document]
Comment (namazgurbanovmdb) [reply src/evergreen/task.py:151]: Its defined under test, not task.

[the remark this answers is not in the document]
Comment (gmishkin) [reply src/evergreen/task.py:151]: Ok, but then you shouldn't add it here, right?

**Review by rongodb — APPROVED** [review]

File: src/evergreen/task.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (namazgurbanovmdb) [reply src/evergreen/task.py:151]: You are right. It already exists under Test. Deleting this PR

**Pull request closed without merging**